[2022-11-13 22:53:26,615] logger_utils.py->print_log line:20 [INFO]----------------获取鉴权码-appid为空接口测试开始----------------
[2022-11-13 22:53:26,615] logger_utils.py->print_log line:20 [INFO]请求方式：None
[2022-11-13 22:53:26,617] logger_utils.py->print_log line:20 [INFO]请求地址：https://api.weixin.qq.com/cgi-bin/token
[2022-11-13 22:53:26,617] logger_utils.py->print_log line:20 [INFO]请求params参数：{'grant_type': 'client_credential', 'appid': '', 'secret': 'e9a29dfa3920020ebee3675bcc9c2cf8'}
[2022-11-13 22:53:26,790] logger_utils.py->print_log line:20 [INFO]正则表达式"access_token":"(.*?)"可能有误，或者(反例)接口请求失败，未提取到中间变量
[2022-11-13 22:53:26,791] logger_utils.py->print_log line:20 [INFO]jsonpath表达式$..expires_in可能有误，或者(反例)接口请求失败，未提取到中间变量
[2022-11-13 22:53:26,791] logger_utils.py->print_log line:20 [INFO]预期结果：[{'codes': {'status_code': 200}}, {'contains': '$ddt{assert_str}'}]
[2022-11-13 22:53:26,791] logger_utils.py->print_log line:20 [INFO]实际结果：{'errcode': 41002, 'errmsg': 'appid missing rid: 637104e6-72640990-12b7efcb'},status_code:200
[2022-11-13 22:53:26,792] logger_utils.py->print_log line:20 [INFO]状态码断言
[2022-11-13 22:53:26,792] logger_utils.py->print_log line:20 [INFO]包含断言
[2022-11-13 22:53:26,792] logger_utils.py->print_log line:20 [INFO]contains断言失败：返回的结果中没有$ddt{assert_str}
[2022-11-13 22:53:26,792] logger_utils.py->print_log line:20 [INFO]获取鉴权码-appid为空业务断言失败
[2022-11-13 22:53:26,793] logger_utils.py->error_log line:15 [ERROR]requests_utils模块standard_yaml_testcase方法报错：Traceback (most recent call last):
  File "D:\Python_project\ms-jiekou\commons\requests_util.py", line 74, in standard_yaml_testcase
    assert 1 == 2, print_log("{}业务断言失败".format(caseinfo["name"]))
AssertionError: None

[2022-11-13 22:53:26,932] logger_utils.py->print_log line:20 [INFO]----------------获取鉴权码-secret为空接口测试开始----------------
[2022-11-13 22:53:26,932] logger_utils.py->print_log line:20 [INFO]请求方式：None
[2022-11-13 22:53:26,933] logger_utils.py->print_log line:20 [INFO]请求地址：https://api.weixin.qq.com/cgi-bin/token
[2022-11-13 22:53:26,933] logger_utils.py->print_log line:20 [INFO]请求params参数：{'grant_type': 'client_credential', 'appid': 'wxf393fadf3268a5d5', 'secret': ''}
[2022-11-13 22:53:27,025] logger_utils.py->print_log line:20 [INFO]正则表达式"access_token":"(.*?)"可能有误，或者(反例)接口请求失败，未提取到中间变量
[2022-11-13 22:53:27,026] logger_utils.py->print_log line:20 [INFO]jsonpath表达式$..expires_in可能有误，或者(反例)接口请求失败，未提取到中间变量
[2022-11-13 22:53:27,026] logger_utils.py->print_log line:20 [INFO]预期结果：[{'codes': {'status_code': 200}}, {'contains': '$ddt{assert_str}'}]
[2022-11-13 22:53:27,026] logger_utils.py->print_log line:20 [INFO]实际结果：{'errcode': 41004, 'errmsg': 'appsecret missing rid: 637104e6-3e810462-46711dd7'},status_code:200
[2022-11-13 22:53:27,026] logger_utils.py->print_log line:20 [INFO]状态码断言
[2022-11-13 22:53:27,026] logger_utils.py->print_log line:20 [INFO]包含断言
[2022-11-13 22:53:27,026] logger_utils.py->print_log line:20 [INFO]contains断言失败：返回的结果中没有$ddt{assert_str}
[2022-11-13 22:53:27,026] logger_utils.py->print_log line:20 [INFO]获取鉴权码-secret为空业务断言失败
[2022-11-13 22:53:27,027] logger_utils.py->error_log line:15 [ERROR]requests_utils模块standard_yaml_testcase方法报错：Traceback (most recent call last):
  File "D:\Python_project\ms-jiekou\commons\requests_util.py", line 74, in standard_yaml_testcase
    assert 1 == 2, print_log("{}业务断言失败".format(caseinfo["name"]))
AssertionError: None

